### ISSUE TITLE
Please add text download option for saved posts or comments

### ISSUE THREAD

**Issue by emk2203** [shadowmoose/RedditDownloader#163]
### Is your feature request related to a problem? Please describe.
When you use a very loose definition of media, I would like to see comment or post text included. This can be optional.

### Describe the solution you'd like:
Please add an optional text download of the saved comments and posts.

**Comment by nguyenanhtuan1008** [comment]
Yes. I hope this feature will be adding soon. Sometimes I want to read the text along with photos or videos. If there is an option to save text .txt with the same file name of the photo/video will be very nice.

**Comment by Z0pyrus** [comment]
yes me too
